pkg/config/config.go

```diff
@@ -845,3 +836,23 @@ func rootlessConfigPath() (string, error) {
 
 	return filepath.Join(home, UserOverrideContainersConfig), nil
 }
+
+func stringsEq(a, b []string) bool {
+
+	// If one is nil, the other must also be nil.
+	if (a == nil) != (b == nil) {
```

Comment:
We don't need the `nil` checks since `len(nil)` is 0. Would speed things up a bit,